src/Scriban/Scriban.csproj

```diff
@@ -9,80 +9,37 @@
     <BuildNumber>005</BuildNumber>
     <VersionSuffix Condition="'$(VersionSuffix)' != '' AND '$(BuildNumber)' != ''">$(VersionSuffix)-$(BuildNumber)</VersionSuffix>
     <Authors>Alexandre Mutel</Authors>
-    <TargetFrameworks>net35;net40;net45;net471;netstandard1.1;netstandard1.3;netstandard2.0;uap10.0;netcoreapp2.0</TargetFrameworks>
+    <TargetFrameworks>net35;net40;net45;netstandard1.1;netstandard1.3;netstandard2.0</TargetFrameworks>
     <AssemblyName>Scriban</AssemblyName>
     <PackageId>Scriban</PackageId>
     <PackageId Condition="'$(SignAssembly)' == 'true'">Scriban.Signed</PackageId>
     <PackageTags>templating;handlebars;liquid</PackageTags>
     <PackageReleaseNotes>https://github.com/lunet-io/scriban/blob/master/changelog.md</PackageReleaseNotes>
     <PackageIconUrl>https://raw.githubusercontent.com/lunet-io/scriban/master/img/scriban.png</PackageIconUrl>
     <PackageProjectUrl>https://github.com/lunet-io/scriban</PackageProjectUrl>
-    <PackageLicenseUrl>https://github.com/lunet-io/scriban/blob/master/license.txt</PackageLicenseUrl>
-    <NetStandardImplicitPackageVersion Condition=" '$(TargetFramework)' == 'netstandard1.1' ">1.6.1</NetStandardImplicitPackageVersion>
```

Comment:
What was that for? Should I have kept it?

Reply:
This should be kept to make sure that the underlying version doesn't work

Reply:
> to make sure that the underlying version doesn't work

I don't understand. I have no problem to remove it, but there must be a reason to explicitly the version of an infrastructure package. Were there problems in the past without this tag?

##### As a sidenote, AFAIK the version of this package does not have any relation with the version of .NET Standard the project targets.

Reply:
Can't remember exactly when, but I had this problem back in the days when they did upgrade the target version behind, so please, leave it as it is, it makes the build less machine/install dependent

Reply:
Done.